charts/refinery/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: refinery
 description: Chart to deploy Honeycomb Refinery
 type: application
-version: 2.18.2
-appVersion: 2.9.7
+version: 2.19.0
```

Comment:
🤔 In the past, we have bumped the major version of the chart when the app major version changes. My first inclination is to continue doing that even though we know this chart will work with a 3.0.0 app image. If user's values YAML for config uses deprecated-and-removed options, their clusters will fail to start.